src/main/java/io/supertokens/session/Session.java

```diff
@@ -132,11 +135,11 @@ public static SessionInformationHolder createNewSession(TenantIdentifier tenantI
                                                             @Nonnull JsonObject userDataInJWT,
                                                             @Nonnull JsonObject userDataInDatabase,
                                                             boolean enableAntiCsrf, AccessToken.VERSION version,
-                                                            boolean useStaticKey)
+                                                            boolean useStaticKey, SemVer semVer)
```

Comment:
Instead of passing semVer here, pass a boolean that indicates whether to check the user tenant or not.

Reply:
done